kani-driver/src/args.rs

```diff
@@ -150,6 +152,9 @@ pub struct KaniArgs {
     /// Specify the value used for loop unwinding for the specified harness in CBMC
     #[arg(long, requires("harness"))]
     pub unwind: Option<u32>,
+    /// Specify the CBMC solver to use. Overrides the harness `solver` attribute.
+    #[arg(long, value_parser = clap::builder::PossibleValuesParser::new(CbmcSolver::VARIANTS))]
+    pub solver: Option<String>,
```

Comment:
I was expecting this to be `Option<Solver>`, and the annotation to allow only a set of literals (associated to solvers) and not strings (e.g., `#[kani::solver(kissat)]`)

Reply:
Changed to `Option<CbmcSolver>`.